docs/faq.rst

```diff
@@ -64,6 +64,15 @@ environment, and will be set to ``True`` when building on RTD::
     Woo
     {% endif %}
 
+My project requires different settings than those available under Admin
+--------------------------------------------------------------------------
+
+Read the Docs offers additional flag settings. To enable these settings,
```

Comment:
Maybe just `settings` instead of `flag settings`?

Reply:
Also, maybe adding some short context on how we use these settings, like: "additional settings to change..."

Reply:
I have rephrased the sentence.